elastic/kibana
# [Synthetics UI] Extract enable/disable functionality from component for reuse

## Summary

There are going to be numerous places where we wish to allow our components to enable/disable a monitor. This patch will extract that functionality to a hook that we can simply call, so components can focus solely on presentation.

## Testing

- load up a few monitors using the synthetics service
- go to the management page, disable your monitor
- navigate to the overview grid, observe that your monitor is disabled
- go back to management, re-enable it
- navigate to the overview grid, observe that your monitor is re-enabled
